src/utils.rs

```diff
@@ -76,16 +76,20 @@ fn get_folder_names(folder: &BTreeMap<OsString, FileTree>) -> Vec<String> {
 #[inline]
 /// "yes" or "no" input dialog, with the option to specify a recommended answer (yes = true, no = false)
 pub fn input_yes_or_no(prompt: &str, recommended: Option<bool>) -> ScaffoldResult<bool> {
-    let yes_recommended = (recommended == Some(true))
-        .then_some("(recommended)")
-        .unwrap_or_default();
-    let no_recommended = (recommended == Some(false))
-        .then_some("(recommended)")
-        .unwrap_or_default();
+    let yes_recommended = if recommended == Some(true) {
```

Comment:
Was that a clippy thing to change the `then_some`s to `if else`s?

Reply:
Yes. I always found the other hard to read but now Clippy agrees with me